.gitignore

```diff
@@ -1,9 +1,5 @@
-/profile/BaseProfiles
-/*.zip
-/*.jar
-/input/images/Thumbs.db
 /input-cache/*
-temp
-template
-output
+/input/*
 
+build/input/*
+SUSHI-GENERATED-FILES.md
```

Comment:
Why is this in .gitignore?

Reply:
Because it is essentially a log of which files were genereated /copied by SUSHI from which source to which target.

Reply:
But at the end, you have the `SUSHI-GENERATED-FILES.md` checked in, so it shouldn't be in the .gitignore.

Reply:
OK, it also shouldn't be checked in 🙈

Reply:
Removed it.